packages/react/src/components/Storage/FileUploader/types.ts

```diff
@@ -101,19 +78,25 @@ export interface TrackerProps {
   onStartEdit: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;
 }
 
-interface FileStatus extends Partial<FileStateProps> {
+export interface FileStatus extends Partial<FileStateProps> {
   percentage?: number;
   uploadTask?: UploadTask;
   fileErrors?: string;
+  name?: string;
+  file?: File;
 }
 
 export type FileStatuses = FileStatus[];
 
+type StatesOfFiles =
```

Comment:
What about just `FileState` so it aligns nicely with `fileState: FileState`?

Reply:
Done

Reply:
#fc15c01